src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Statute.java

```diff
@@ -14,19 +11,25 @@
 @Getter
 @Setter
 @NoArgsConstructor
+@AllArgsConstructor
 public class Statute {
```

Comment:
When we pull in the ORDS Statue, can we map to our column names. For example this seems cleaner

public record Statute(
    string Id, 
    string ActCode, 
    string SectionText, 
    string SubsectionText,
    string ParagraphText, 
    string SubparagraphText, 
    string Code, 
    string ShortDescriptionText, 
    string DescriptionText
);